monero-outreach-docs/translations/pt-br/storyboard/blockchain_gender_pt-br.md

```diff
@@ -0,0 +1,78 @@
+
+Mais recentemente, estive envolvida nos bastidores como gerente / coordenadora de operações em um programa do YouTube chamado “[Monero Talk w / @chowbungaman](https://www.youtube.com/channel/UC3Hx81QYLoEQkm3vyl4N4eQ)”. É um programa sobre todas as coisas relacionadas ao Monero. Queremos preencher a lacuna sobre informações envolvendo o Monero.
+
+Como atualmente tudo é mais voltado ao Bitcoin, decidimos criar este canal para pessoas que estão interessadas no Monero e até agora ele está indo muito bem. Tivemos grandes visitantes e nos disseram que ele é muito informativo. Estamos tentando expor  todas as informações relacionadas ao Monero, um episódio de cada vez.
+
+**O que atraiu você em particular sobre o Monero que fez você querer se envolver em um nível mais substancial?**
+
+Ele tem todos os parâmetros que o Bitcoin está tentando alcançar: é privado, não rastreável e fungível. Essas são as qualidades perfeitas de uma criptomoeda criada para promover sua meta de descentralização e que poderia ser usada como uma moeda digital peer-to-peer.
+
+O que eu também adoro é que não está sobrecarregado com marketing falso, como o da ICO de outras altcoins, e que tudo é promovido pelas pessoas da comunidade. O crescimento é muito mais orgânico e mais forte porque os seguidores não foram convencidos por uma campanha de marketing ou por um discurso retórico.
+
+**Como você encontrou trabalho no projeto Monero e na comunidade em geral?**
+
+Tem sido um espaço muito interessante que tem sido muito acolhedor e prestativo. O que eu gosto é que todos estão dispostos a ajudar uns aos outros em prol do objetivo, que é a descentralização.
+
+**Você pode compartilhar algumas idéias sobre suas experiências de ser uma mulher na indústria de criptografia / blockchain? Há alguma vantagem ou desvantagem que você tenha experienciado?**
+
+Eu não experenciei nenhuma desvantagem em ser uma mulher no espaço da criptografia - ele foi acolhedor e ótimo até agora. Eu sinto que nesse ambiente você tem a capacidade e a oportunidade de contribuir da maneira que quiser, não importa o gênero. O ambiente é muito open source e se você tiver uma sugestão, ideia ou qualquer coisa que melhore um projeto, você será ouvido.
+
+No final do dia, tem a ver com quais competências e ideias você traz para a mesa, e se isso tiver um impacto positivo no projeto, isso será reconhecido. Eu entendo que as mulheres querem ajudar a aumentar a participação das mulheres nesse espaço, mas, na minha opinião, focar em promover os eventos “Mulheres em Blockchain / Cripto” tende a fazer o oposto do que se pretendia e causa um ambiente mais segregado e divisor. Esses eventos devem ser mais uma colaboração de mulheres e homens compartilhando ideias para ajudar a melhorar seu projeto específico.
+
+**Por que você acha que a participação de mulheres na indústria de criptografia é relativamente menor? Como você acha que essas barreiras podem ser quebradas?**
```

Comment:
"Superadas" instead of "quebradas" so it matches the terms used in the last paragraph.